frontend/src/app/utils/utils.ts

```diff
@@ -28,3 +28,9 @@ export const showLoading =<T>(isLoading: BehaviorSubject<boolean>, promise: Prom
         return result;
     });
 };
+
+/**
+ * returns a function that always returns `value`
+ */
+export const to = (value?: any) =>
+    () => value;
```

Comment:
Why?

Reply:
I initially used it in several places and then removed all but one, so that's why it wasn't not defined inline.

Reply:
I don't know whether Underscore or Lodash is used in I-analyzer, but this is `_.constant`. It is very general-purpose so I would say there is no shame in having it as a standalone function.